File: leetcode/easy/1470/1470.js
/**
 * Runtime: 92 ms, faster than 35.85% of JavaScript online submissions for Shuffle the Array.
 * Memory Usage: 40.7 MB, less than 5.02% of JavaScript online submissions for Shuffle the Array.
 */
var shuffle = function(nums, n) {
    let result = [];
    
    for(let i=0; i<n; ++i) {
        result.push(nums[i]);
        result.push(nums[n+i]);
    }
    
    return result;
};

File: leetcode/easy/1470/1470.rb
# Runtime: 48 ms, faster than 89.60% of Ruby online submissions for Shuffle the Array.
# Memory Usage: 210.7 MB, less than 61.85% 

def shuffle(nums, n)
    result = []
    
    1.upto(n) do |i|
        result << nums[i-1];
        result << nums[n+(i-1)]
    end
    result
end
